proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,536 @@
+   internationaler Kommuniikation.
+4. Der Verein respektiert die Privatsphäre der Nutzer der angebotenen
+   Web-Dienste und der veröffentlichten Software und erlaubt ihnen die anonyme
+   Teilnahme oder mit vollständiger Identifizierung.
+5. Die Vereinigung verpflichtet sich zur Transparenz und sammelt keine Daten
+   der Nutzer ohne deren ausdrückliche Kenntnisnahme und Zustimmung.
+6. In Fällen, in denen die Dienste des Vereins von einem Drittanbieter
+   bereitgestellt werden die eine separate Benutzeranmeldung erfordern, gelten
+   die Datenschutzbestimmungen des Drittanbieters.
+
+## 14. Salvatorische Klausel
+
+Sollten einzelne Bestimmungen dieser Satzung ganz oder teilweise unwirksam oder
+undurchführbar sein oder werden, so berührt dies nicht die Gültigkeit der
+Satzung insgesamt.
+
+## 15. Inkrafttreten
+
+Diese Satzung tritt mit Beschlussfassung der Mitgliederversammlung in Kraft.
+
+
+## English Draft
+
+## Articles of Association - Mixxx e.V. -
+
+As a non-profit corporation incorporated under German law, Mixxx e.V. is
+required to declare its intentions and by-laws in a formal "Articles of
+Association" document. The original and legally binding version is the German
+version. The following translation is for information only.
+
+## Clarification of Terminology
+
+1. Free and Open Source Software (FOSS) refers to computer programs that
+   respect the freedom of the user and the community. It grants everyone the
+   right to run, study, modify and redistribute the software. This promotes
+   the common good as everyone can use the software and adapt it to their
+   needs. FOSS is particularly beneficial for schools, artists and musicians
+   with low budgets or incomes, as it gives them access to powerful tools for
+   education, creativity and productivity without financial barriers.
+2. Developing free and open source software involves research, compilation of
+   principles and concepts, and their testing through programming and testing
+   of free software.
+3. "Mixxx" is a DJ software developed as free and open source software and
+   available to the public free of charge. It allows users to mix music tracks
+   in real-time, providing features such as beatmatching, crossfading, and
+   various sound effects.
+
+## 1. Association Name, Registered Office, Financial Year
+
+1. The Association is registered as Mixxx e.V.
+2. The registered offices are located in .... . If no fixed office is
+   established, it will be managed from the residence of the Board member
+   acting as general manager.
+3. It is listed under the appropriate register in the District Court’s
+   Register of Associations.
+4. The financial year is the calendar year.
+
+## 2. Purpose of the Association
+
+1. The Association’s purpose is to promote education and occupational training
+   as well as art and culture. It provides music artists with the basic
+   building blocks for individual performance setups on a free and open source
```

Comment:
'music artists'  makes it narrow. There could be video / spoken word /  graphical artists that use Mixxx 
for there performance to.
Maybe just 'artists' ?

Reply:
I have fiddled quite long with this sentences. We need a not too bulky definition that covers: 
* live DJs
* Beedroom DJs 
* Recorded mix creators.  
* Dance teachers 
* Radio DJs

I would argue that video / spoken word / graphical artists are not at the focus of Mixxx e.V. this does not mean that we actively exclude them, but all things there need are also covered by use cases of the of "music artists ... individual performance" 

The alternative to list the roles explicit covers the risk that it is treated as a definite list, which I like to avoid as well. 

But anyway, I am open to alternative proposals.

Reply:
fi "artists who need reproduction of audio"

Reply:
artists .. audio performance setups should work